template/src/main.js

```diff
@@ -450,12 +450,20 @@ function init () {
 
       // show api
       $('.show-api').click(function () {
+        const selectedVersion = $('#version strong').html();
```

Comment:
while there is nothing inherently wrong with jquery, I prefer the new code to use vanilla js where possible, so this line should be replaced by: `document.getElementById('version').innerText.trim()`. Also, this selector will not break if the "strong" is removed later on (as it might very well be because that should be a job for css, and not old school html tags).

There is no need to try and rewrite everything in vanilla, but a bit less of jquery here and there doesn't hurt ;)